### DISCUSSION TITLE
easy editing of presets

### DISCUSSION THREAD

**Discussion by gitrimius** [shanteacontrols/OpenDeck#78] [Q&A]
Hi,
I have a lot of analog inputs (64) and buttons (8) etc... When editing a new preset I have to remake all the settings I made in previous presets. Sometimes I only want to change the settings of some analog inputs, but leave the rest unchanged. 
Is there a method/trick (by editing the sysex backup file for example) to base a presets on the previous preset/or a certain preset?
Thanks!

**Comment by paradajz** [comment]
The only way to do that is to modify the backup file, yes. I suggest to wait for new release since my hardware testing setup discovered some issues with backup/restore. Will be out today most likely.
